doc/subsystems/disk/sdhc.rst

```diff
@@ -0,0 +1,56 @@
+.. _SDHC_disks:
+
+SDHC disks
+##########
+
+Zephyr includes support for connecting an SDHC card via the SPI bus.
+This can be used with Zephyr's built-in filesystem support to read and
+write FAT formatted cards.
+
+The system has been tested with cards from Samsung, SanDisk, and 4V
+with sizes from 2 GiB to 32 GiB in single partition mode.  Higher
+capacity cards should also work but haven't been tested.  Please let
+us know if they work!
+
+MMC and SDSC (<= 2 GiB) cards are not supported and will be ignored.
+
+The system does not support inserting or removing cards while the
```

Comment:
This seems rather important to point out, so let's change to:
```
note:: The system does not support inserting or removing cards while the
   system is running.   The cards must be present at boot and must not be
   removed.  This may be fixed in future releases.
  
   FAT filesystems are not power safe and may become corrupted
   if power is lost or the card is removed.
```